internal/driver/device.go

```diff
@@ -92,39 +92,50 @@ func (dev *Device) StopStreaming() {
 }
 
 // SetFps updates the fps on the device side of the service. Note that this won't update the rtsp output stream fps
-func (dev *Device) SetFps(device *usbdevice.Device, fpsNumerator uint32, fpsDenominator uint32) (string, error) {
+func (dev *Device) SetFrameRate(usbdevice *usbdevice.Device, fpsNumerator uint32, fpsDenominator uint32) (string, error) {
```

Comment:
fps Num/Denom args should be renamed to match the same as what you renamed them in the calling code.